### PR TITLE
chore(lerna): lerna deploy scripts

### PR DESCRIPTION
**Pull request by jgolden17**
Add lerna deployment scripts

### CODE REVIEW

**Comment by jgolden17** [comment]
> I might output a message (to stderr?) if the commit message is malformed, that way the user is alerted that the deploy failed.

The idea is that it should be a noop on any other commit, not that the deployment should fail

File: package.json [hunk 1]
```diff
@@ -19,7 +19,10 @@
     "lint": "eslint --ext js,jsx,md ./",
     "test:git-history": "commitlint --from origin/master --to HEAD",
     "test:lockfile": "node lockFileLint.js",
-    "posttest": "npm run test:lockfile"
+    "posttest": "npm run test:lockfile",
+    "lerna:version": "lerna version",
```
Comment (nellyk) [inline package.json:23]: Would you like to document this in a release.md file similar to this https://github.com/americanexpress/one-app-cli/blob/master/CONTRIBUTING.md#creating-a-one-app-cli-new-release

Comment (jgolden17) [reply package.json:23]: Yeah, we should add a CONTRIBUTING.md file in a subsequent PR.  I'll add an issue for it.

Comment (jgolden17) [reply package.json:23]: #51

**Pull request merged**